### PR TITLE
SAK-48329 Message is sent even after removing scheduling date

### PR DESCRIPTION
**Pull request by susowrc**
Message is sent even after removing scheduling date

https://sakaiproject.atlassian.net/browse/SAK-48329

### CODE REVIEW

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java [hunk 1]
```diff
@@ -1898,16 +1898,20 @@ public String processPvtMsgSaveDraft() {
...
     }else{
-    	dMsg =constructMessage(true, null) ;
+	    dMsg =constructMessage(true, null) ;
     }
     dMsg.setDraft(Boolean.TRUE);
     dMsg.setDeleted(Boolean.FALSE);
     dMsg.setExternalEmail(booleanEmailOut);
     dMsg.setScheduler(booleanSchedulerSend);
-    if(booleanSchedulerSend)
+    if(booleanSchedulerSend){
```
Comment (mpellicer) [inline msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java:1910]: There's a small spacing issue, but you can also use this:

dMsg.setScheduledDate(booleanSchedulerSend ? openDate : null);

**Review by mpellicer — CHANGES_REQUESTED** [review]
Looks good to me, just some minor spacing issues.

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java [hunk 2]
```diff
@@ -1898,16 +1898,20 @@ public String processPvtMsgSaveDraft() {
 
     PrivateMessage dMsg = null;
     if(getDetailMsg() != null && getDetailMsg().getMsg() != null && getDetailMsg().getMsg().getDraft()){
-    	dMsg =constructMessage(true, getDetailMsg().getMsg()) ;
+	    dMsg =constructMessage(true, getDetailMsg().getMsg()) ;
```
Comment (mpellicer) [inline msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java:1901]: space before the semicolon and after the equal

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java (hunk 1 above)
Comment (jesusmmp) [reply msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java]: Done!

File: msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java (hunk 2 above)
Comment (jesusmmp) [reply msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java]: Done!

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Welcome abroad @susowrc and good job ! Thanks for the help @jesusmmp

**Comment by mpellicer** [comment]
@susowrc Do you see yourself listed here?

http://licensing.apereo.org/completed-clas

**Review by jesusmmp — APPROVED** [review]

**Pull request merged**